src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/wwwroot/lib/bootstrap/.bower.json

```diff
@@ -29,17 +29,18 @@
     "test-infra"
   ],
   "dependencies": {
-    "jquery": "1.9.1 - 3"
+    "jquery": "1.9.1 - 3",
+    "bootstrap": "4.1.1"
   },
-  "version": "3.3.7",
-  "_release": "3.3.7",
+  "version": "4.1.1",
```

Comment:
this is a different version than linked as production version (4.0.0) used above:
https://github.com/aspnet/templating/pull/595/files#diff-6bbdca69c7ee830ddc10bdf0d31f2941R64
The problem is that MS cdn lags updates:
https://docs.microsoft.com/en-us/aspnet/ajax/cdn/overview#Bootstrap_Releases_on_the_CDN_14

Reply:
Good point, they updated the CDN, so I just moved everything up to 4.1.1.

Reply:
Nice to hear that! Glad I could help somehow